src/java.base/share/classes/sun/security/pkcs10/PKCS10Attributes.java

```diff
@@ -94,7 +93,7 @@ public PKCS10Attributes(DerInputStream in) throws IOException {
      * @param out the OutputStream to marshal the contents to.
      * @exception IOException on encoding errors.
      */
-    public void encode(OutputStream out) throws IOException {
+    public void encode(DerOutputStream out) throws IOException {
```

Comment:
Add @Override

Reply:
I think you are missing `@Override` on some of the other `encode` and `derEncode` methods.